src/commands/dev/lib/dev-server.ts

```diff
@@ -184,26 +185,84 @@ export default class DevServer {
     );
   }
 
-  async send404(res: http.ServerResponse): Promise<void> {
+  async send404(
+    req: http.IncomingMessage,
+    res: http.ServerResponse,
+    nowRequestId: string
+  ): Promise<void> {
     return this.sendError(
+      req,
       res,
+      nowRequestId,
       'FILE_NOT_FOUND',
       'The page could not be found',
       404
     );
   }
 
   async sendError(
+    req: http.IncomingMessage,
```

Comment:
Unused param

Reply:
I'm anticipating using it to detect the `Accept` request header and send a text / json / html error response, so I'll just leave it for now.